packages/cli/src/commands/db/revert.ts

```diff
@@ -33,7 +33,10 @@ export async function main(
 	}
 
 	if (!lastMigration.down) {
-		logger.error('The last migration was irreversible and cannot be reverted.');
+		const message = lastMigration.name
+			? `The last migration "${lastMigration.name}" was irreversible and cannot be reverted.`
+			: 'The last migration was irreversible and cannot be reverted.';
```

Comment:
`cannot be reverted` is the definition of `irreversible` so I'm not a fan of this message.

As a user, what I'd like to know is if the revert I requested is being cancelled because the last migration is irreversible.

Reply:
I guess we have 2 scenarios we must take into account:

1. It was an implementation of `IrreversibleMigration` and hence we cannot revert or
2. as explained above, the user downgraded n8n and the migration file could not be found

In the latter, we must say that the user should try running the migration using the updated version of n8n since the migration could not be found in the current installation in order to be reverted.

Reply:
I can change the message and remove the duplication. 

> As a user, what I'd like to know is if the revert I requested is being cancelled because the last migration is irreversible.

Yes. I don't quite know if this is a request to change something or just an observation 🤔

Reply:
> if this is a request to change something or just an observation 

I don't find this clear enough, but I don't feel strongly about it, up to you.